src/components/User/SystemAssets/components/AssetManager.vue

```diff
@@ -227,6 +227,14 @@
       </div>
 
       <template v-if="isShownAdvanced">
+        <div class="app__form-row">
+          <tick-field
+            class="app__form-field"
+            v-model="asset.creatorDetails.isCoinpayments"
+            label="Use Coinpayments"
```

Comment:
Label describes nothing. You can also add a title to provide more info about what the checkbox does

Reply:
ok